### PR TITLE
[one-cmds] Revise one-build section error message

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise one-build section error message.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
This has effect to solve some problem with my vscode cause of odd number of `'` character.

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/one-cmds/tests/one-build_neg_003.test [hunk 1]
```diff
@@ -21,7 +21,7 @@ filename="${filename_ext%.*}"
 
 trap_err_onexit()
 {
-  if grep -q "'one-build' section is required in configuraion file" "${filename}.log"; then
+  if grep -q "section is required in configuraion file" "${filename}.log"; then
```
Comment (seanshpark) [inline compiler/one-cmds/tests/one-build_neg_003.test:24]: using `[]` seems to work as regex pattern match and `[section]` fails to work

Comment (mhs4670go) [reply compiler/one-cmds/tests/one-build_neg_003.test:24]: Maybe `\[one-build\]`? Or is it backslash that makes error?

Comment (seanshpark) [reply compiler/one-cmds/tests/one-build_neg_003.test:24]: > Maybe `\[one-build\]` ?

I'll try this one

> Or is it backslash that makes error?

Yes `[]` acts as regex pattern so `[one-build]` tries something and fails.

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: compiler/one-cmds/tests/one-build_neg_003.test (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/tests/one-build_neg_003.test:24]: `\[one-build\]` this works :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**